tilelang/language/eager/__init__.py

```
@@ -1,2 +1,15 @@
 from .builder import prim_func, macro, PrimFunc, JITFunc, Ref, const, annotate_compile_flags, annotate_pass_configs  # noqa: F401
-from ..dtypes import *
+from ..dtypes import *  # noqa: F401,F403
+from ..dtypes import __all__ as _dtypes_all
+
+__all__ = (
```

Ruff (0.15.21)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)